### DISCUSSION TITLE
Undo manager - Not working properly in removing colors

### DISCUSSION THREAD

**Discussion by DhanaDSP1120** [GrapesJS/grapesjs#4690] [Q&A]
Undo manager - Not working properly, when we try to apply and undo the background color then made click outside the block, then the removed color will appears

It occurs on version 0.19.4 to till latest , its working fine on below versions

Reproduce links

Working version :
https://jsfiddle.net/0vpbLmxy/

Not working version :
https://jsfiddle.net/q3cgwa5m/1/

Demo :
![grapesjs undo issue](https://user-images.githubusercontent.com/54067384/197989568-5b29f18e-cf93-4cd6-a6ad-bec0ddd283ad.gif)

**Comment by artf** [comment]
https://github.com/artf/grapesjs/issues/4691
